configs/components/hiera.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/hiera.git", "ref": "3047016da65d12ff7e316db17e40adec9a080090"}
+{"url": "git://github.com/puppetlabs/hiera.git", "ref": "refs/tags/3.1.0"}
```

Comment:
The tag is off `fc7614d6ba81845757ba7318269fad2b2da08da3`. `3047016da65d12ff7e316db17e40adec9a080090` doesn't seem to exist in the hiera repo... So I'd like some validation on this tag. It also looks like the latest hiera tag is `3.1.5`.

cc/ @kylog